tests/factories.py

```diff
@@ -437,25 +422,13 @@ def _create(cls, *args, **kwargs):
         return fork
 
 
-class PointerFactory(ModularOdmFactory):
-    class Meta:
-        model = Pointer
-    node = SubFactory(NodeFactory)
-
-
 class NodeLogFactory(ModularOdmFactory):
```

Comment:
Eventually, these factories should also be removed. Usages should be replaced with the ones in `osf_tests/factories.py`